### PR TITLE
Added getUserInfo endpoint

### PR DESCRIPTION
**Pull request by MRuy**
This endpoint provides the following data about the user:

* userID
* userName
* minutesSaved
* segmentCount
* viewCount

Could also contain warnings for users with "bad" segment submissions

### CODE REVIEW

**Review by ajayyy — APPROVED** [review]

File: test/cases/postWarning.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    before(() => {
+      db.exec("INSERT INTO vipUsers (userID) VALUES ('" + getHash("warning-vip") + "')");
+    });
+  
+    it('Should update the database version when starting the application', (done) => {
+        let version = db.prepare('get', 'SELECT key, value FROM config where key = ?', ['version']).value;
+        if (version > 3) done();
+        else done('Version isn\'t greater than 3. Version is ' + version);
+    });
+
```
Comment (ajayyy) [inline test/cases/postWarning.js:16]:
```suggestion
```

**Review by ajayyy — CHANGES_REQUESTED** [review]
Can the warnings cause votes and submissions to fail with the warning as the error message?

File: test/cases/postWarning.js [hunk 2]
```diff
@@ -8,12 +8,6 @@ describe('postWarning', () => {
       db.exec("INSERT INTO vipUsers (userID) VALUES ('" + getHash("warning-vip") + "')");
     });
   
-    it('Should update the database version when starting the application', (done) => {
```
Comment (Joe-Dowd) [inline test/cases/postWarning.js]: Why remove the test case?

Comment (ajayyy) [reply test/cases/postWarning.js]: This test case is already in the dedicated `dbUpgrade.js`

**Review by ajayyy — APPROVED** [review]

**Pull request merged**